go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -78,7 +78,31 @@ func (tm *TabletManager) RestoreData(ctx context.Context, logger logutil.Logger,
 	if tm.Cnf == nil {
 		return fmt.Errorf("cannot perform restore without my.cnf, please restart vttablet with a my.cnf file specified")
 	}
-	return tm.restoreDataLocked(ctx, logger, waitForBackupInterval, deleteBeforeRestore)
+	// Tell Orchestrator we're stopped on purpose for some Vitess task.
```

Comment:
I'm surprised that we didn't already have this change. But in any case, calling it out as being related to additional orc calls.

Reply:
The change is new. It was backported only two weeks ago: https://github.com/vitessio/vitess/pull/8063

However, I'm surprised we haven't run into issues due to this.

Reply:
I agree. I would have to check the orc code to be sure, but maybe because when we're restoring, `mysql` isn't reachable so orc just ... mostly ignores the tablet?